php/class-media.php

```diff
@@ -1985,6 +2028,25 @@ public function can_filter_out_local() {
 		return $can;
 	}
 
+	/**
+	 * Fix the PDF resource type.
+	 *
+	 * @link https://cloudinary.com/cookbook/convert_pdf_to_jpg
+	 *
+	 * @param string $type          The default type.
+	 * @param int    $attachment_id The attachment ID.
+	 *
+	 * @return string
+	 **/
+	public function pdf_resource_type( $type, $attachment_id ) {
```

Comment:
@pereirinha - This is a problem. PDF cannot be used as an image, since it will break it as a downloadable. look at 
`Media::is_preview_only()`